src/main/java/net/minestom/arena/group/GroupCommand.java

```diff
@@ -15,6 +21,33 @@ public GroupCommand() {
         super("group");
         setCondition(CommandUtils::lobbyOnly);
 
+        addSyntax((sender, context) -> {
+            if (sender instanceof Player player) {
```

Comment:
The player check is unnecessary. As already made by `CommandUtils::lobbyOnly`

Reply:
Ahh I just did so because the one below did as well. I'll change it

Reply:
Will have to be fixed too

Reply:
I'll change those as well